README.md

```diff
@@ -1,4 +1,4 @@
-kiwix-html5
+kiwix-JS
```

Comment:
Maybe with an uppercase on the K, to follow https://github.com/kiwix/kiwix-html5/issues/248#issuecomment-309490013